controllers/dspipeline_params.go

```diff
@@ -389,11 +392,10 @@ func (p *DSPAParams) SetupObjectParams(ctx context.Context, dsp *dspa.DataScienc
 
 }
 
-func (p *DSPAParams) SetupMLMD(ctx context.Context, dsp *dspa.DataSciencePipelinesApplication, client client.Client, log logr.Logger) error {
+func (p *DSPAParams) SetupMLMD(dsp *dspa.DataSciencePipelinesApplication) error {
```

Comment:
Looks like you changed function parameters. Is that intentional?

Reply:
fwiw i checked this and the parameters are unused in this function, so this is *probably* actually an improvement (unless you want to be a stickler about homogeneous signatures for every SetupXYZ() function)

Reply:
^ exactly.